### ISSUE TITLE
Training strateges

### ISSUE THREAD

**Issue by mikhailkonyk** [jingyunx/Deformation-Flow-Based-Two-stream-Network#3]
Hello! How to properly change learning rate during training? I tried to do that, then val loss stagnates. I stopped training, changed lr, and start training again with new weights, but after that val loss also stagnates.. What's wrong? Thank you.

**Comment by daqiudi** [comment]
> Hello! How to properly change learning rate during training? I tried to do that, then val loss stagnates. I stopped training, changed lr, and start training again with new weights, but after that val loss also stagnates.. What's wrong? Thank you.

Hi, what data set did you train on
